src/layout/layout.js

```diff
@@ -433,10 +433,9 @@
 
       // Keep an eye on screen size, and add/remove auxiliary class for styling
       // of small screens.
-      this.screenSizeMediaQuery_ = this.matchMedia_(
-          /** @type {string} */ (this.Constant_.MAX_WIDTH));
-      this.screenSizeMediaQuery_.addListener(this.screenSizeHandler_.bind(this));
-      this.screenSizeHandler_();
+      this.screenSizeMediaQuery_ = window.materialLayoutScreenWidthMediaQuery
+      this.screenSizeMediaQuery_.addEventListener('change', screenSizeHandler);
```

Comment:
There's no reason to add a handler for each MaterialLayout, since the handler deals with all of them.
Just add a static boolean property on MaterialLayout to track whether the eventhandler has been registered or not.

Reply:
Agreed, and I think we can do it without the bool